### ISSUE TITLE
training details

### ISSUE THREAD

**Issue by Laidawang** [TheMistoAI/MistoLine#6]
I tried your model and it worked very well. Can you share the details of the training? For example, batch size, learning rate.

**Comment by Laidawang** [comment]
So does this mean no explanation will be given? XD

**Comment by BNP1111** [comment]
yes
